poc_mobile_verifier/src/heartbeats.rs

```diff
@@ -209,18 +209,12 @@ impl Heartbeat {
             return Ok(false);
         }
 
-        sqlx::query("DELETE FROM heartbeats WHERE cbsd_id = $1 AND hotspot_key != $2 ")
```

Comment:
does this still do what the original change intended it to do? if we're not clearing the table from records of the last heartbeats that were registered to a hotspot aren't we just building up two sets of records in parallel here (one radio attached to two hotspots)? couldn't a game just keep switching them back and forth between hotspots throughout the day?

Reply:
I believe it should because we change the constraint from being unique (hotspot_key, cbsd_id) to just cbsd_id (see the new migration)

Reply:
the insert statement now matches only on cbsd_id and updates hotspot_key on conflict